components/provisioning/org.wso2.carbon.identity.provisioning/src/main/java/org/wso2/carbon/identity/provisioning/ProvisioningUtil.java

```diff
@@ -48,12 +48,14 @@ private ProvisioningUtil() {
                                               String claimUri, String userStoreDomainName) {
 
         List<String> claimValues = new ArrayList<>();
-        for (Map.Entry<ClaimMapping, List<String>> entry : attributeMap.entrySet()) {
-            ClaimMapping mapping = entry.getKey();
-            if (mapping.getLocalClaim() != null
-                    && claimUri.equals(mapping.getLocalClaim().getClaimUri())) {
-                claimValues = entry.getValue();
-                break;
+        if (attributeMap != null) {
```

Comment:
Shall we use Array Utils or CollectionUtils to check whether this is empty or null ?